core/src/main/scala/org/bitcoins/core/crypto/ExtKeyVersion.scala

```diff
@@ -31,4 +15,20 @@ object ExtKeyVersion {
 
   def apply(bytes: ByteVector): Option[ExtKeyVersion] =
     all.find(_.bytes == bytes)
+
+  case object MainNetPub extends ExtKeyVersion {
+    override def bytes = ByteVector(0x04, 0x88, 0xb2, 0x1E)
```

Comment:
Nit: I think we should change these to 

```
override val bytes = ...
```

This means the byte vector doesn't get re-computed every time.

Reply:
Another nit: I believe we should mark these as `final` according to https://nrinaudo.github.io/scala-best-practices/adts/final_case_objects.html